### PR TITLE
Dịch "natural-language-inference-bert_vn" - Phần 3

### LINKED ISSUE #2653: Dịch "natural-language-inference-bert_vn" - Phần 3
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md#L263-L333

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by nguyenthaibinh**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #2653

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy thảo luận thêm ở thẻ Discussions của dự án.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review khi Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 1]
```diff
@@ -268,7 +268,9 @@ In common implementations of BERT fine-tuning, only the parameters of the output
 All the parameters of the pretrained BERT encoder (`net.encoder`) and the hidden layer of the additional MLP (net.hidden) will be fine-tuned.
 -->
 
-*dịch đoạn phía trên*
+Sau đây, mô hình BERT tiền huấn luyện `bert` được đưa vào thể hiện `net` của `BERTClassifier` cho ứng dụng hạ nguồn.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:271]:
```suggestion
Sau đây, mô hình BERT đã tiền huấn luyện `bert` được đưa vào thực thể `net` của lớp `BERTClassifier` cho ứng dụng xuôi dòng.
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 2]
```diff
@@ -268,7 +268,9 @@ In common implementations of BERT fine-tuning, only the parameters of the output
...
+Trong các phần cài đặt chung của tinh chỉnh BERT, chỉ các tham số của lớp đầu ra của perception đa tầng bổ sung (`net.output`) mới được học từ đầu.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:272]:
```suggestion
Thường khi lập trình tinh chỉnh BERT, chỉ các tham số của lớp đầu ra của perception đa tầng bổ sung (`net.output`) mới được học từ đầu.
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 3]
```diff
@@ -268,7 +268,9 @@ In common implementations of BERT fine-tuning, only the parameters of the output
...
+Còn tất cả các tham số của bộ mã hóa BERT tiền huấn luyện (`net.encoder`) và lớp ẩn của perception đa tầng bổ sung (net.hidden) sẽ được tinh chỉnh.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:273]:
```suggestion
Còn tất cả các tham số của bộ mã hóa BERT đã tiền huấn luyện (`net.encoder`) và lớp ẩn của perception đa tầng bổ sung (net.hidden) sẽ được tinh chỉnh.
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 4]
```diff
@@ -285,7 +287,11 @@ These two loss functions are irrelevant to fine-tuning downstream applications,
 `MaskLM` and `NextSentencePred` are not updated (staled) when BERT is fine-tuned.
 -->
 
-*dịch đoạn phía trên*
+Nhớ lại rằng trong :numref:`sec_bert`, cả 2 lớp` MaskLM` và lớp `NextSentencePred` đều có các tham số trong perceptron đa tầng mà chúng đã sử dụng.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:290]:
```suggestion
Nhớ lại rằng trong :numref:`sec_bert`, cả 2 lớp` MaskLM` và lớp `NextSentencePred` đều có các tham số trong perceptron đa tầng mà chúng sử dụng.
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 5]
```diff
@@ -285,7 +287,11 @@ These two loss functions are irrelevant to fine-tuning downstream applications,
...
+Các tham số này là một phần của các tham số trong mô hình BERT tiền huấn luyện `bert`, và do đó là một phần của các tham số trong `net`.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:291]:
```suggestion
Các tham số này là một phần của các tham số trong mô hình BERT đã tiền huấn luyện `bert`, và do đó là một phần của các tham số trong `net`.
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 6]
```diff
@@ -285,7 +287,11 @@ These two loss functions are irrelevant to fine-tuning downstream applications,
...
+Tuy nhiên, các tham số này chỉ được dùng để tính toán mất mát của mô hình ngôn ngữ có mặt nạ và mất mát khi dự đoán câu tiếp theo trong quá trình tiền huấn luyện.
+Hai hàm mất mát này không liên quan đến việc tinh chỉnh trong các ứng dụng hạ nguồn, do đó các tham số của perceptron đa tầng dùng trong
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:293]:
```suggestion
Hai hàm mất mát này không liên quan đến việc tinh chỉnh trong các ứng dụng xuôi dòng, do đó các tham số của perceptron đa tầng dùng trong
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 7]
```diff
@@ -285,7 +287,11 @@ These two loss functions are irrelevant to fine-tuning downstream applications,
...
+`MaskLM` và` NextSentencePred` không được cập nhật khi BERT được tinh chỉnh.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:294]:
```suggestion
`MaskLM` và` NextSentencePred` không được cập nhật khi tinh chỉnh BERT.
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 8]
```diff
@@ -294,7 +300,9 @@ We use this function to train and evaluate the model `net` using the training se
 Due to the limited computational resources, the training and testing accuracy can be further improved: we leave its discussions in the exercises.
 -->
 
-*dịch đoạn phía trên*
+Để cho phép sử dụng các tham số với gradient cũ, ta đặt cờ `ignore_stale_grad = True` trong hàm `step` của `d2l.train_batch_ch13`.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:303]:
```suggestion
Để cho phép sử dụng các tham số với gradient không được cập nhật, ta đặt cờ `ignore_stale_grad = True` trong hàm `step` của `d2l.train_batch_ch13`.
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 9]
```diff
@@ -313,7 +321,9 @@ d2l.train_ch13(net, train_iter, test_iter, loss, trainer, num_epochs, devices,
 Parameters that are only related to pretraining loss will not be updated during fine-tuning. 
 -->
 
-*dịch đoạn phía trên*
+* Chúng ta có thể tinh chỉnh mô hình BERT tiền huấn luyện cho các ứng dụng hạ nguồn, chẳng hạn như suy luận ngôn ngữ tự nhiên trên tập dữ liệu SNLI.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:324]:
```suggestion
* Chúng ta có thể tinh chỉnh mô hình BERT đã tiền huấn luyện cho các ứng dụng xuôi dòng, chẳng hạn như suy luận ngôn ngữ tự nhiên trên tập dữ liệu SNLI.
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 10]
```diff
@@ -313,7 +321,9 @@ d2l.train_ch13(net, train_iter, test_iter, loss, trainer, num_epochs, devices,
...
+* Trong quá trình tinh chỉnh, mô hình BERT trở thành một phần của mô hình ứng dụng hạ nguồn.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:325]:
```suggestion
* Trong quá trình tinh chỉnh, mô hình BERT trở thành một phần của mô hình ứng dụng xuôi dòng.
```

File: chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md [hunk 11]
```diff
@@ -327,7 +337,12 @@ By increasing fine-tuning epochs (and possibly tuning other hyperparameters), ca
 Compare this pair truncation method and the one used in the `SNLIBERTDataset` class. What are their pros and cons?
 -->
 
-*dịch đoạn phía trên*
+1. Hãy tinh chỉnh một mô hình BERT tiền huấn luyện lớn hơn, có kích thước tương đương với mô hình cơ sở BERT ban đầu, nếu tài nguyên tính toán của bạn cho phép.
```
Comment (cuongvng) [inline chapter_natural-language-processing-applications/natural-language-inference-bert_vn.md:340]:
```suggestion
1. Hãy tinh chỉnh một mô hình BERT tiền huấn luyện lớn hơn, có kích thước tương đương với mô hình BERT cơ sở ban đầu, nếu tài nguyên tính toán của bạn cho phép.
```

**Review by cuongvng — APPROVED** [review]
